hedera-mirror-rest/api/v1/openapi.yml

```
@@ -4424,7 +4426,7 @@ components:
         type: string
         pattern: ^((eq|gt|gte|lt|lte):)?\d{1,19}?$
     timestampQueryParam:
-      description: The consensus timestamp in seconds.nanoseconds format with an optional comparison operator
+      description: The consensus timestamp as a Unix timestamp in seconds.nanoseconds format with an optional comparison operator
```

Same comment for other similar areas.

```suggestion
      description: The consensus timestamp as a Unix timestamp in seconds.nanoseconds format with an optional comparison operator. See [unixtimestamp.com](https://www.unixtimestamp.com/) for a simple way to convert a date to the 'seconds' part of the Unix time.
```

---

Got it, I have added the link